src/compiler/bundle/ext-transforms-plugin.ts

```diff
@@ -42,20 +62,36 @@ export const extTransformsPlugin = (
         return null;
       }
 
+      /**
+       * Make sure compiler context has a registered worker. The interface suggests that it
+       * potentially can be undefined, therefore check for it here.
+       */
+      if (!compilerCtx.worker) {
+        return null;
+      }
+
       // The `id` here was possibly previously updated using
       // `serializeImportPath` to annotate the filepath with various metadata
       // serialized to query-params. If that was done for this particular `id`
       // then the `data` prop will not be null.
       const { data } = parseImportPath(id);
 
       if (data != null) {
+        let cmpStyles: ComponentStyleMap | undefined = undefined;
         let cmp: d.ComponentCompilerMeta | undefined = undefined;
         const filePath = normalizeFsPath(id);
         const code = await compilerCtx.fs.readFile(filePath);
         if (typeof code !== 'string') {
           return null;
         }
 
+        /**
+         * add file to watch list if it is outside of the `srcDir` config path
+         */
+        if (config.watch && (id.startsWith('/') || id.startsWith('.')) && !id.startsWith(config.srcDir)) {
```

Comment:
I think I'd personally lean toward a config option that allows devs to specify additional files/dirs to watch (like a glob pattern array). But, if I'm the only one then this works too!